views/Feed/Feed.jsx

```diff
@@ -56,6 +51,14 @@ export default class FeedView extends React.Component {
     await this.loadData();
 
     this.setState({ loading: false });
+
+    //refresh data when returning from CreatePost
+    this.props.navigation.addListener(
```

Comment:
Not a huge fan of the way this is done, but good enough for now.
I think ideally the child components should take a function as a prop, which updates the state of this component with the changes from the child. This would prevent a full load, resulting in a snappier feel and reduced network usage.

Again, I think the changes here outweigh that though so lets just throw that on the back burner and roll with this.